maxim/logger/livekit/utils.py

```
@@ -8,6 +12,65 @@
     get_session_store,
 )
+from ...scribe import scribe
+
+
+class SameThreadExecutor(Executor):
+    """
+    A mock executor that runs submitted callables on the same thread, synchronously.
+    Mimics the interface of concurrent.futures.Executor.
+    """
+
+    def __init__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)